Source/WebCore/Modules/mediasource/ManagedMediaSource.idl

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2023 Apple Inc. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are
+ * met:
+ *
+ *     * Redistributions of source code must retain the above copyright
+ * notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following disclaimer
+ * in the documentation and/or other materials provided with the
+ * distribution.
+ *     * Neither the name of Google Inc. nor the names of its
+ * contributors may be used to endorse or promote products derived from
+ * this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+enum BufferingPolicy {
+    "low",
+    "medium",
+    "high"
+};
+
+enum PreferredQuality {
+    "low",
+    "medium",
+    "high"
+};
+
+[
+    ActiveDOMObject,
+    Conditional=MANAGED_MEDIA_SOURCE,
+    EnabledBySetting=ManagedMediaSourceEnabled,
+    Exposed=Window
+] interface ManagedMediaSource : MediaSource {
+    [CallWith=CurrentScriptExecutionContext] constructor();
+
+    readonly attribute BufferingPolicy buffering;
+    readonly attribute PreferredQuality quality;
+
+    attribute EventHandler onstartstreaming;
+    attribute EventHandler onendstreaming;
+
+    [CallWith=CurrentScriptExecutionContext] static boolean isTypeSupported (DOMString type);
```

Comment:
```suggestion
    [CallWith=CurrentScriptExecutionContext] static boolean isTypeSupported(DOMString type);
```

Reply:
This is the syntax as used in MediaSource.idl and the spec one (always a space)